pkg/actions/event.go

```diff
@@ -20,6 +20,7 @@ type EventInfo struct {
 	CommitMessage  string            `json:"commit_message,omitempty"`
 	Committer      string            `json:"committer,omitempty"`
 	CommitMetadata map[string]string `json:"commit_metadata,omitempty"`
+	MergeSource    string            `json:"merge_source,omitempty"`
```

Comment:
I was having trouble understanding the difference between MergeSource and SourceRef. I think documenting these fields will greatly improve readability

Reply:
Documented in the HookRecord struct and part of the invoke

Reply:
I think it's better to document it on the struct itself rather than where it is being used

Reply:
HookRecord is the structure that is used when we trigger hooks.

Reply:
Sorry I thought you meant on the usage